_licenses/cern-ohl-p-2.0.txt

```diff
@@ -0,0 +1,216 @@
+---
+title: CERN Open Hardware Licence Version 2 - Permissive
+spdx-id: CERN-OHL-P-2.0
+nickname: CERN OHL v2 Permissive
+
+description: The CERN OHL was drafted with hardware in mind.  On top of licensing copyright like software and documentation licenses, it also licenses patents.  This “Permissive” variant allows people to take your code, relicense it and use it without any obligation to distribute the sources when they ship a product; it does have an attribution requirement.
+
+how: Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file.
+
+note: CERN <a href="https://ohwr.org/project/cernohl/wikis/uploads/8a6b5d01f71c207c49493e4d114d61e6/cern_ohl_p_v2_howto.pdf">recommends</a> adding the following information in the source files: copyright and license notice, a disclaimer of warranties and a source location. You can also include the source location in a part of the design which will be visible in the end product.
```

Comment:
Test failures are coming from the `:` in this value. The whole string needs to be quoted if it contains a colon. I guess the href value could be single quoted, or switch link to markdown `[]()` syntax.

Reply:
I just spotted the YAML error and shifted it all to use indented multiline text which I think should work

Reply:
Different YAML error this time, only in the -S variant. Sorry for the sloppy lack of linting. Linted it to check this time